TexSoup/tokens.py

```diff
@@ -303,7 +303,7 @@ def tokenize_punctuation_command_name(text, prev=None):
     if text.peek(-1) and text.peek(-1).category == CC.Escape:
         for point in PUNCTUATION_COMMANDS:
             if text.peek((0, len(point))) == point:
-                result = text.forward(len(point) + 1)
+                result = text.forward(len(point))
```

Comment:
credit hint in https://github.com/alvinwan/TexSoup/issues/111#issuecomment-708664125